### DISCUSSION TITLE
2Fort Players Don't Know What's About to Hit Them

### DISCUSSION THREAD

**Discussion by Kacey2k** [dborodin836/TF2-GPTChatBot#98] [General]
I dont know why you did it.
I dont know if it should have been done.
But you have done it.

Going to join a 2fort server and give players permanent trust issues with this.

**Comment by teufortressIndustries** [comment]
Lmao
Edit: I'm actually scared. [What have you done?](https://wiki.teamfortress.com/w/images/d/d8/Spy_sf13_magic_reac04.wav)
